### ISSUE TITLE
operator_usage_whitespace autocorrect inconsistencies

### ISSUE THREAD

**Issue by SixFiveSoftware** [realm/SwiftLint#3321]
### New Issue Checklist

- [x] Updated SwiftLint to the latest version
- [x] I searched for [existing GitHub issues](https://github.com/realm/SwiftLint/issues)

### Describe the bug

When running SwiftLint autocorrect, the operator_usage_whitespace rule will fix equal sign column alignment for some right-hand side operands, but not some literals. For example:

```
let color           = UIColor.black // triggers violation
let six               = 6 // triggers violation
let foo              = "foo" // does not trigger violation, but should
let array           = [3.14] // does not trigger violation, but should
```

So, after auto-correct, I'm left with:

```
let color = UIColor.black
let six = 6
let foo              = "foo"
let array           = [3.14]
```

##### Complete output when running SwiftLint, including the stack trace and command used

```bash
$ swiftlint lint
```

for properly triggered violations:
```
warning: Operator Usage Whitespace Violation: Operators should be surrounded by a single whitespace when they are being used. (operator_usage_whitespace)
```

for the String/Array literals listed above, no output is given (i.e. linting succeeds)

### Environment

* SwiftLint version (run `swiftlint version` to be sure)? 0.39.2
* Installation method used (Homebrew, CocoaPods, building from source, etc)? CocoaPods
* Paste your configuration file:

```yml
opt_in_rules:
...
  - operator_usage_whitespace
...
```

* Are you using [nested configurations](https://github.com/realm/SwiftLint#nested-configurations)? No
  If so, paste their relative paths and respective contents.
* Which Xcode version are you using (check `xcodebuild -version`)? 11.3.1 (11C505)
* Do you have a sample that shows the issue? Run `echo "[string here]" | swiftlint lint --no-cache --use-stdin --enable-all-rules`
  to quickly test if your example is really demonstrating the issue. If your example is more
  complex, you can use `swiftlint lint --path [file here] --no-cache --enable-all-rules`.

```swift
let foo              = "foo" // does not trigger violation, but should
let array           = [3.14] // does not trigger violation, but should
```

**Comment by serejahh** [comment]
PR is merged, so the issue should be closed I guess. @SixFiveSoftware could you update SwiftLint to verify if it works for you now?

**Comment by SixFiveSoftware** [comment]
> PR is merged, so the issue should be closed I guess. @SixFiveSoftware could you update SwiftLint to verify if it works for you now?

Yes, it worked! Thank you!
